hack/golangci.yaml.in

```diff
@@ -185,84 +191,135 @@ linters:
+        - "-S1008"   # TODO(fix) Simplify returning boolean expression
+        - "-S1009"   # TODO(fix) Omit redundant nil check on slices, maps, and channels
+        - "-S1011"   # TODO(fix) Use a single append to concatenate two slices
+        - "-S1012"   # TODO(fix) Replace time.Now().Sub(x) with time.Since(x)
+        - "-S1016"   # TODO(fix) Use a type conversion instead of manually copying struct fields
+        - "-S1017"   # TODO(fix) Replace manual trimming with strings.TrimPrefix
+        - "-S1019"   # TODO(fix) Simplify make call by omitting redundant arguments
+        - "-S1021"   # TODO(fix) Merge variable declaration and assignment
+        - "-S1023"   # TODO(fix) Omit redundant control flow
+        - "-S1024"   # TODO(fix) Replace x.Sub(time.Now()) with time.Until(x)
+        - "-S1025"   # TODO(fix) Don’t use fmt.Sprintf("%s", x) unnecessarily
+        - "-S1028"   # TODO(fix) Simplify error construction with fmt.Errorf
+        - "-S1030"   # TODO(fix) Use bytes.Buffer.String or bytes.Buffer.Bytes
+        - "-S1034"   # TODO(fix) Use result of type assertion to simplify cases
+        - "-S1036"   # TODO(fix) Unnecessary guard around map access
+        - "-S1038"   # TODO(fix) Unnecessarily complex way of printing formatted string
+        - "-S1039"   # TODO(fix) Unnecessary use of fmt.Sprint
+        - "-S1040"   # TODO(fix) Type assertion to current type
+        - "-SA1006"  # TODO(fix) Printf with dynamic first argument and no further arguments
+        - "-SA1019"  # TODO(fix) Using a deprecated function, variable, constant or field
+        - "-SA2002"  # TODO(fix) Called testing.T.FailNow or SkipNow in a goroutine, which isn’t allowed
+        - "-SA4006"  # TODO(fix) A value assigned to a variable is never read before being overwritten. Forgotten error check or dead code?
+        - "-ST1000"  # TODO(fix) Incorrect or missing package comment
+        - "-ST1001"  # TODO(fix) Dot imports are discouraged
+        - "-ST1003"  # TODO(fix) Poorly chosen identifier
+        - "-ST1005"  # TODO(fix) Incorrectly formatted error string
+        - "-ST1006"  # TODO(fix) Poorly chosen receiver name
+        - "-ST1008"  # TODO(fix) A function’s error value should be its last return value
+        - "-ST1011"  # TODO(fix) Poorly chosen name for variable of type time.Duration
+        - "-ST1012"  # TODO(fix) Poorly chosen name for error variable
+        - "-ST1013"  # TODO(fix) Should use constants for HTTP error codes, not magic numbers
+        - "-ST1016"  # TODO(fix) Use consistent method receiver names
+        - "-ST1017"  # TODO(fix) Don’t use Yoda conditions
+        - "-ST1020"  # TODO(fix) The documentation of an exported function should start with the function’s name
+        - "-ST1023"  # TODO(fix) Redundant type in variable declaration
+        {{- end}}
+    testifylint:
+      enable-all: true
+      {{- if not .Hints }}
+      disable:  # TODO: remove each disabled rule and fix it
```

Comment:
Same here: let's remove the `TODO: `.

Improvements to the code failing these checks are possible, but they need to be justified in the PR with "this is better because..." nor with "fix linter issue".